src/components/Icon/index.js

```diff
@@ -34,13 +41,34 @@ class Icon extends PureComponent {
         const width = this.props.small ? variables.iconSizeSmall : this.props.width;
         const height = this.props.small ? variables.iconSizeSmall : this.props.height;
         const IconToRender = this.props.src;
-        return (
+
+        const rendered = (
```

Comment:
```suggestion
        const icon = (
```
NAB: I think icon is better name here.